### ISSUE TITLE
Microsoft.Build.Tasks.v4.0.dll not found

### ISSUE THREAD

**Issue by PhilPJL** [aspnet/RoslynCodeDomProvider#53]
When building on a TFS 2015 build server with VS 2013, 2015 and 2017 installed, this fails:

`  <UsingTask TaskName="CheckIfVBCSCompilerWillOverride" TaskFactory="CodeTaskFactory" AssemblyFile="$(MSBuildToolsPath)\Microsoft.Build.Tasks.v4.0.dll">`

Copying the .v4.0.dll from the .NET framework folder into C:\Program Files (x86)\MSBuild\14.0\Bin\amd64 fixes the issue.

**Comment by harrisse** [comment]
I believe I'm hitting this as well:
```
.\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets(97,5): error MSB4062: The "EnsureBindingRedirects" task could not be loaded from the assembly .\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.Tasks.dll. Could not load file or assembly 'Microsoft.Build.Utilities.v4.0, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.
```

**Comment by harrisse** [comment]
Nevermind, my package missing is different -- Microsoft.Build.Utilities.v4.0 but still potentially related.

**Comment by chbohm** [comment]
> I believe I'm hitting this as well:
> 
> ```
> .\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.targets(97,5): error MSB4062: The "EnsureBindingRedirects" task could not be loaded from the assembly .\packages\Microsoft.Bcl.Build.1.0.21\build\Microsoft.Bcl.Build.Tasks.dll. Could not load file or assembly 'Microsoft.Build.Utilities.v4.0, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a'. The system cannot find the file specified. Confirm that the <UsingTask> declaration is correct, that the assembly and all its dependencies are available, and that the task contains a public class that implements Microsoft.Build.Framework.ITask.
> ```

Hi harrisse.... I am having exacly the same error message. Could you find a fix for this? It has passed one year.... I hope you remember!

**Comment by StephenMolloy** [comment]
I believe #92 should fix this

**Comment by StephenMolloy** [comment]
Should be fixed by #92
